game/src/assets/html/settings.html

```diff
@@ -1,5 +1,5 @@
-<h1>Coming soon...</h1>
-
+<h1>Settings</h1>
+<input data-value = offlineMode id="settingsCheck" type="checkbox" > Enable Offline Mode
```

Comment:
the value of the data-value attribute should be inside double quotes: `data-value="offlineMode"`